docs/cudf/source/groupby.md

```diff
@@ -166,3 +166,8 @@ a
 2 3  <NA>  <NA>  <NA>
   4     4     5     9
 ```
+
+## Notes
```

Comment:
Let's move this bit to the section on grouping and add an example showing the difference in behaviour.

Rather than using a heading, let's put this in a warning box. One easy way to do that in markdown:

```
> :warning: **Important**: Be very careful here!
> more warning text
```

---

Another way is to use RST (ReStructuredText) to write the warning: see [here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html), and embed the RST in this markdown document using the approach mentioned [here](https://recommonmark.readthedocs.io/en/latest/auto_structify.html#embed-restructuredtext). I think this latter approach also requires small changes to `setup.py` mentioned [here](https://recommonmark.readthedocs.io/en/latest/auto_structify.html#configuring-autostructify). Not sure if it's worth the pain/effort.